### ISSUE TITLE
Menu Settings Button Accessibility Label Is Not Labeled Correctly.

### ISSUE THREAD

**Issue by JB184351** [woocommerce/woocommerce-ios#6595]
This Settings Button is just labeled Button in the Menu tab of the app. Would recommend labeling it as Settings.

![image](https://user-images.githubusercontent.com/36495451/162237299-d670407e-a07b-4b8d-beca-bcf837b9e587.png)
